digits/model/tasks/caffe_train.py

```diff
@@ -787,10 +788,24 @@ def iteration_to_epoch(self, it):
 
     @override
     def task_arguments(self, resources, env):
+        if platform.system() == 'Windows':
+            network = caffe_pb2.NetParameter()
+            network.CopyFrom(self.network)
+            if any([layer.type == 'Python' for layer in network.layer]):
+                win_python_layer_gpu_id = None
+                if 'gpus' in resources:
+                    n_gpus = len(resources['gpus'])
+                    if n_gpus > 1:
+                        raise Exception('Please select single GPU when running in Windows with Python layer.')
+                    elif n_gpus == 1:
+                        win_python_layer_gpu_id = resources['gpus'][0][0]
+                win_python_layer_args = self.pycaffe_args(win_python_layer_gpu_id)
+                return win_python_layer_args
+
         args = [config_value('caffe_root')['executable'],
-                'train',
-                '--solver=%s' % self.path(self.solver_file),
-                ]
+            'train',
+            '--solver=%s' % self.path(self.solver_file),
+            ]
```

Comment:
Seems my IDE changed them.  I will change its settings, and revert those spaces back.